aries-site/src/examples/components/maskedinput/MaskedIPAddressExample.js

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import { Form, FormField, MaskedInput } from 'grommet';
+
+const IPv4ElementExp = /^[0-1][0-9][0-9]$|^2[0-4][0-9]$|^25[0-5]$|^[0-9][0-9]$|^[0-9]$/;
+
+export const MaskedIPAddressExample = () => (
+  <Form>
+    <FormField
+      label="IP Address"
+      htmlFor="masked-ip-address"
+      name="maskedIpAddress"
+    >
+      <MaskedInput
+        name="maskedIpAddress"
+        id="masked-ip-address"
+        mask={[
```

Comment:
Suggestion: separate the mask logic from the MaskInput to aid readability and cognition.

In the previous examples, the mask was its own variable and object, then referenced by the MaskedInput component. I find those examples easier to read because it allows me to compartmentalize and separate the mask logic from the MaskInput configuration.

Reply:
Agreed for all of these, I will separate out to separate variable.